app/controllers/admins/ws_panel_controller.rb

```diff
@@ -145,7 +159,8 @@ def edit_dispatcher
         end
         if @dispatcher.save
           send_message :xnotice, 'message' => 'Changes successfully saved'
-          send_message :get_edit_dispatcher_data, dispatcher_data
+          dispatcher_data[:author] = current_admin.id
+          broadcast_message :get_edit_dispatcher_data, dispatcher_data
         else
           send_message :xnotice, 'message' => 'Changing dispatcher: ERROR'
```

Comment:
By the way, I think that it will be more informative to return the text of the actual errors instead of just `ERROR`.